types/assert-equal-jsx/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for assert-equal-jsx 1.0
+// Project: https://github.com/thejameskyle/assert-equal-jsx
+// Definitions by: Josh Toft <https://pack.io>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as React from 'react';
+
+export type SanitizeFunc = (str: string) => string;
```

Comment:
So the reason I used a function here was due to the lint failure that occurs if you don't:

```
ERROR: 10:5  prefer-method-signature  Use a method signature instead of a property signature of function type.
```

It felt weird when I was writing it, but what should we do to make this pass and also not be a trivial type? Any suggestions here? I'm assuming we want it to pass the linter.